### PR TITLE
Update coloryze for working with custom ranges

### PR DESCRIPTION
**Pull request by vit134**
Update coloryze for working with custom ranges
Create demo for this

### CODE REVIEW

File: examples/main/index.js [hunk 1]
```diff
@@ -2,6 +2,10 @@ import '../../src/Polygonmap';
 import points from '../data/bikeparking-moscow.geojson';
 import polygons from '../data/moscow-mo.geojson';
 
+pointsPyaterochka.features.forEach((element) => {
```
Comment (ruslankhh) [inline examples/main/index.js]: Remove it.

File: src/utils/defaultMapper.js [hunk 2]
```diff
@@ -7,7 +7,10 @@
  */
 const defaultMapper = function (feature) {
     feature.options = {
-        fillColor: this.colorize.getColor(feature.properties.pointsCount)
+        fillColor: this.colorize.getColor(feature.properties.pointsCount),
+        fillOpacity: this.options.get('colorOpacity'),
```
Comment (ruslankhh) [inline src/utils/defaultMapper.js]: Maybe, all the same, we'll rename this option.
```js
{
    ...
    fillOpacity: this.options.get('fillOpacity'),
    ...
}
```

Comment (vit134) [reply src/utils/defaultMapper.js]: Not agree. Then it will not be consistent with other settings in Api associated with the color of the polygons

Comment (ruslankhh) [reply src/utils/defaultMapper.js]: Okay.

**Pull request merged**